docs/icons.stories.js

```diff
@@ -38,6 +39,9 @@ stories.add('icon', () => {
     'is-small': 'is-small',
     'is-medium': 'is-medium',
     'is-large': 'is-large',
+    'is-half': 'is-half',
```

Comment:
thanks for your PR 😍

42-43 only apply to the heart icon, so they're more an icon than a variation.

Can we add `is-half` and `is-empty` after line 20 as normal icons?

Reply:
Thanks for the feedback, I'll take care of that. Sorry it's been a little jacked up, I'm used to using git in TFS for version control.